cypress/e2e/DateEditor.spec.ts

```diff
@@ -79,15 +118,18 @@ describe('Date Editor', () => {
 
     it('should render date, time (24 format) and timezone inputs by default', () => {
       openPage();
-
       selectors.getDateInput().should('be.visible').should('have.value', '');
       selectors
         .getTimeInput()
         .should('be.visible')
         .should('have.attr', 'date-time-type', '24')
         .should('have.attr', 'placeholder', '00:00')
         .should('have.value', '00:00');
-      selectors.getTimezoneInput().should('be.visible').should('have.value', '+00:00');
+      // TODO find a better way
```

Comment:
out of curiosity: why did we have to change the whole date behavior in this file?

Reply:
cy.clock doesn't work in iframe :(

Reply:
as a work around I adding some functions to handle the time stuff, not ideal but the best option I could come up with. Happy to hear suggestions is you have some